src/Latte/Runtime/FilterInfo.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * This file is part of the Latte (https://latte.nette.org)
+ * Copyright (c) 2008 David Grudl (https://davidgrudl.com)
+ */
+
+namespace Latte\Runtime;
+
+use Latte;
+
+
+/**
+ * Filter runtime info
+ */
+class FilterInfo
+{
+	use Latte\Strict;
+
+	/** @var string|NULL */
+	public $contentType;
+
+
+	public function __construct($contentType = NULL)
```

Comment:
is it neccesary to create new instance every time? Imho this could be made into [flyweight](https://en.wikipedia.org/wiki/Flyweight_pattern).